### PR TITLE
basics/alias-strings: typos, headings, link to unicode gem

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: basics/alias-strings.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ different threads. As `string` is a slice, parts can be taken out of it without
 allocating memory. The standard function `std.algorithm.splitter`
 for example splits a string by newline without any memory allocations.
 
-Beside the UTF-8 `string` there are two more types:
+Besides the UTF-8 `string` there are two more types:
```
Comment (wilzbach) [inline basics/alias-strings.md]: http://dictionary.cambridge.org/grammar/british-grammar/beside-or-besides

File: basics/alias-strings.md [hunk 2]
```diff
@@ -80,6 +86,7 @@ of reserved symbols. Raw strings can be declared using either backticks (`` `
 
 ### In-depth
 
+- [Unicode gem](gems/unicode)
```
Comment (wilzbach) [inline basics/alias-strings.md]: I wasn't sure where to link to it. We could also link to it within the text?

Comment (ZombineDev) [inline basics/alias-strings.md]: I'm sure that I understand you, but I think that the like is appropriate.

Comment (wilzbach) [inline basics/alias-strings.md]: I realized that we don't link to the new Unicode gem and saying that we could also add a reference within the Unicode paragraph, sth like "for more details see the Unicode gem"

**Pull request merged**